bin/gen_c_header.py

```diff
@@ -0,0 +1,188 @@
+#!/usr/bin/env python3
+
+################################################################################
+#
+# Copyright 2020 OpenHW Group
+# Copyright 2020 Silicon Labs, Inc.
```

Comment:
Should be 2022 or 2023.

And I think @MikeOpenHWGroup said before that the openhw group copyright line isn't really necessary?